README.md

```diff
@@ -1,3 +1,43 @@
-# Castone Project
+# Capstone Project
 
-This is the capstone project github repo
+This is the first capstone project github repo
+
+> In this is a website designed using css, javascript and html based on this design <https://www.behance.net/gallery/29845175/CC-Global-Summit-2015>
+
+## Built With
+
+- Eslint
+- Webhint
+- Styelint
+- CSS/Sass
+
+## Live Demo
```

Comment:
- [ ] Good job for adding video presentation of the project, however kindly endeavor to add in the video description some **salient aspects** of your code

Reply:
Thank you I've added this to the description.